README.md

```diff
@@ -6,30 +6,28 @@ An _experimental_ (Read: not-usable or in anyway done) distributed podcasting pl
 
 Getting started:
 
-Install the vendor dependencies using godep. (https://github.com/golang/dep)
+We use [fresh](https://github.com/pilu/fresh) for hot code reloading with docker-compose.
 
-```sh
-dep ensure
-```
-
-Building a binary:
+Building docker-compose for development:
 
 ```sh
 make
 ```
 
-Building and then running that binary:
+Building and then running docker-compose:
 
 ```sh
 make run
 ```
 
-Running tests:
+Building docker containers for production:
 
 ```sh
-make test
```

Comment:
FIX: Keep `make test` separate from making prod. It encourages people to write and run tests often and at the moment. They should be as fast as possible to run.